bench/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "start": "node index.js",
+    "test": "node index.js"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "benchmark": "2.1.x",
+    "cli-table": "0.3.x",
+    "colors": "1.1.x",
+    "sanctuary-type-classes": "5.2.0"
```

Comment:
Good idea! This should do the trick:

```bash
node --eval '
  var fs = require("fs");
  var pkg = require("./bench/package.json");
  pkg.dependencies["sanctuary-type-classes"] = process.env.VERSION;
  fs.writeFileSync("./bench/package.json",
                   JSON.stringify(pkg, null, 2) + "\n",
                   {encoding: "utf8"});
'
```

Surprisingly, the `require("fs")` isn't actually necessary (but let's not rely on this surprising behaviour).

While you're making this change, please manually update the version above to the latest version. :)